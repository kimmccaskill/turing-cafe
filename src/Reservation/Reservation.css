.res-card {
  background-color: #fff;
  border: solid teal 3px;
  border-radius: 10px;
  color: teal;
  margin: 1%;
  padding: 0 1% 1% 1%;
  min-width: 160px;
}

.cancel-btn {
  background-color: teal;
  color: #fff;
  width: 90%;
}