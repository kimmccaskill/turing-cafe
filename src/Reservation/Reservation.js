import React from 'react';
import './Reservation.css'

const Reservation = ({ details }) => {
  return (
    <article className='res-card'>
      <h1>{details.name}</h1>
      <p>{details.date}</p>
      <p>{details.time} pm</p>
      <p>Number of Guests: {details.number}</p>
      <button className='cancel-btn'>Cancel</button>
    </article>
  )
}

export default Reservation;