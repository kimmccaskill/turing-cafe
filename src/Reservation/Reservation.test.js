import React from 'react';
import Reservation from './Reservation';
import { shallow } from 'enzyme';

describe('Reservation', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
    <Reservation 
      details={[{
          "id": 1,
          "name": "Christie",
          "date": "12/29",
          "time": "7:00",
          "number": 12
    }]}
    />)
  })

  it('should be an instance of Reservation component', () => {
    expect(wrapper).toMatchSnapshot();
  });
})