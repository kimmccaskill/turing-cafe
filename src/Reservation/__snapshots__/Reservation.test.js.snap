// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reservation should be an instance of Reservation component 1`] = `
<article
  className="res-card"
>
  <h1 />
  <p />
  <p>
     pm
  </p>
  <p>
    Number of Guests: 
  </p>
  <button
    className="cancel-btn"
  >
    Cancel
  </button>
</article>
`;
